src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const populationSpans = document.querySelectorAll('.population');
+
+let totalPopulation = 0;
+
+populationSpans.forEach(populationSpan => {
```

Comment:
Consider using `reduce` method to calculate the sum of an array for more concise and readable code.